packages/shared/styles/components/organisms/SfProductCard.scss

```diff
@@ -36,6 +36,7 @@
     line-height: 0;
     text-decoration: none;
     margin: var(--product-card-link-margin, 0);
+    text-align: left
```

Comment:
I am not sure whether this is crucial but there is no `;` at the end of this line. I am mentioning this purely because in previous projects I had situations where this sort of thing would break the project ;)

Reply:
Yes, it is :) Good point